### ISSUE TITLE
Ask for your test dataset

### ISSUE THREAD

**Issue by Fduxiaozhige** [monakhova/starlight_denoising#9]
Hi,

Thanks for sharing this repo. You have done a really good work.

I want to ask for you test dataset. I ran the training code, and the KLD is 0.006, this is far more lower 0.069, i think it may test on the validation or training dataset. Can you tell me how to test on your test set?

Thanks a lot.
czz
